boards/arm/ks_zion/doc/index.rst

```diff
@@ -0,0 +1,128 @@
+.. _ks_zion_board:
+
+
+Kernelspace Zion
+################
+
+.. contents::
+
+
+1. Overview
+===========
+
+The Kernelspace Zion features an ARM Cortex-M4 based STM32F303K8
+MCU with the minal required hardware layout for beginners to get started
+quickly. It's a simple open-hardware  board that can be assembled easily
+at home.
+
+For further info please contact info@kernel-space.org.
```

Comment:
It would be much better if you could have this be a link to a publicly accessible webpage. Right now the added documentation mentions an open hardware board but it's not even showing up on ex. Google (I realize the board might be brand new so I might have simply not been able to find it online!)

Reply:
ok will do, thanks